pkgs/applications/graphics/durdraw/default.nix

```
@@ -0,0 +1,34 @@
+{ lib
+, python3Packages
+, fetchFromGitHub
+, ansilove
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "durdraw";
+  version = "0.20.2";
```

```suggestion
  version = "0.20.2";
  pyproject = true;
```
and maybe add `setuptool` to `nativeBuildInputs`.